WalletWasabi/JsonConverters/ByteArrayJsonConverter.cs

```diff
@@ -29,9 +22,16 @@ public override bool CanConvert(Type objectType)
 		}
 
 		/// <inheritdoc />
-		public override void WriteJson(JsonWriter writer, object value, JsonSerializer serializer)
+		public override void WriteJson(JsonWriter writer, byte[]? value, JsonSerializer serializer)
 		{
-			writer.WriteValue(Convert.ToBase64String((byte[])value));
+			if (value is null)
+			{
+				throw new ArgumentNullException(nameof(value));
```

Comment:
So this is the exception that was thrown before?

Reply:
Yes.

![kimi](https://user-images.githubusercontent.com/52379387/146770682-96a5ece9-7409-4a15-81f7-837e9b8d93c0.PNG)